app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -65,12 +75,27 @@ fun ProfileScreen(
               horizontalArrangement = Arrangement.spacedBy(10.dp, Alignment.CenterHorizontally),
               verticalAlignment = Alignment.Top) {
 
-                // profile placeholder
-                Image(
-                    painter = painterResource(id = R.drawable.profile),
-                    contentDescription = "profile picture",
-                    modifier = Modifier.size(200.dp).testTag("profilePicture"),
-                )
+                // fetch profile picture from firebase
+                val auth = Firebase.auth
+                val user = auth.currentUser
+                val photo = user?.photoUrl
+                val placeholderUri =
+                    "https://picsum.photos/200/300".toUri() // Placeholder image URI
```

Comment:
In my opinion, you should rather store this profile picture locally - in the case of the app not connected to internet, we should be able to display it.

Reply:
I removed the placeholder URI